Content.Shared/Contraband/ContrabandSystem.cs

```diff
@@ -55,10 +57,23 @@ private void OnExamined(Entity<ContrabandComponent> ent, ref ExaminedEvent args)
         using (args.PushGroup(nameof(ContrabandComponent)))
         {
             var severity = _proto.Index(ent.Comp.Severity);
+            var jobList = new List<string>();
             if (severity.ShowDepartments && ent.Comp is { AllowedDepartments: not null })
             {
+                if (ent.Comp is { AllowedJobs: not null })
```

Comment:
What in the case we allow a certain job, but not a department?
Nesting the if conditions like this means it won't display the jobs in that case

Reply:
It worked in testing for the galoshes when I set the allowed departments to an empty list instead of null, but I'll try and find a solution that truly allows the departments field to be null.

Reply:
still unresolved

Reply:
Shit, sorry. I used Github to apply these changes directly from browser, but I think I overwrote them on accident when I committed the yaml changes afterwards. I'll fix em when I get home.

Reply:
I've nested both departments and jobs checks within the existing if section so that either (or even both) can be null without preventing the rest of the function from executing.